src/test/datascience/execution.unit.test.ts

```diff
@@ -1,1135 +0,0 @@
-// Copyright (c) Microsoft Corporation.

```

Comment:
See here https://github.com/microsoft/vscode-jupyter/pull/11673
This PR depends on Removed in separate PR https://github.com/microsoft/vscode-jupyter/pull/11673